go-controller/pkg/cni/helper_linux.go

```diff
@@ -315,6 +323,20 @@ func (pr *PodRequest) ConfigureInterface(namespace string, podName string, ifInf
 		}
 	}
 
+	err = netns.Do(func(hostNS ns.NetNS) error {
+		if _, err := os.Stat("/proc/sys/net/ipv6/conf/all/dad_transmits"); !os.IsNotExist(err) {
+			err = setSysctl("net/ipv6/conf/all/dad_transmits", 0)
```

Comment:
Including `/proc/sys` in the first line but not the second makes it look like one of them is wrong.
Maybe if there aren't going to be other sysctls being set then there's no reason to have a separate `setSysctl` function.

Reply:
good point on the mismatch here.  I added the os.Stat() later.